sujeong/app.js

```diff
@@ -0,0 +1,53 @@
+const http = require('http');
+
+require('dotenv').config();
+const express = require('express');
+const cors = require('cors');
+const morgan = require('morgan');
+
+const {dataSource, DataSource} = require('typeorm');
+const { runInNewContext } = require('vm');
```

Comment:
```suggestion
require('dotenv').config();

const http = require('http');
const express = require('express');
const cors = require('cors');
const morgan = require('morgan');

const { DataSource } = require('typeorm');

```

`const { runInNewContext } = require('vm');` 해당 부분은 어디서 사용해서 호출하신걸까요?

Reply:
영상보면서 하다가 들어간거같아요 지웠습니다!